### ISSUE TITLE
Modal

### ISSUE THREAD

**Issue by ajthyng** [txstate-etc/txstate-components#3]
* Modal should accept any size content inside of it
* Scrim color on a modal should be configurable

**Comment by wickning1** [comment]
- Modal should have the ability (default: on) to lock focus inside it for accessibility.
- Focus should return to where it was before the modal, after the modal is dismissed.
